actioncable/lib/action_cable/connection/faye_client_socket.rb

```diff
@@ -3,9 +3,10 @@
 module ActionCable
   module Connection
     class FayeClientSocket
-      def initialize(env, event_target, stream_event_loop)
+      def initialize(env, event_target, stream_event_loop, protocols: ActionCable::INTERNAL[:protocols])
```

Comment:
I think we can drop the default here for protocols -- the option to pass in a different array is already presented to users in `Connection::WebSocket#initialize`